src/components/InputBox/InputBox.scss

```
@@ -0,0 +1,33 @@
+.inputBox {
+  padding: 10px 0 20px;
+  width: 400px;
+
+  .inputTitle {
+    font-size: 13px;
+    width: 100%;
+    font-weight: 700;
+  }
+  .inputItem {
+    width: 100%;
+    border-bottom: solid rgb(233, 233, 233) 1px;
+    font-size: 13px;
+
+    .input {
+      width: 100%;
+      padding: 12px 30px 8px 0;
+      background-color: transparent;
+      border-bottom: solid black 1px;
+      border-radius: 0;
+      border: none;
+      padding: 8px 0 px 0;
+      resize: none;
+    }
+    input:focus {
+      outline: none;
+    }
```

부모 선택자(&) 활용하시면 이렇게 작성할 수도 있습니다.
```suggestion
    .input {
      width: 100%;
      padding: 12px 30px 8px 0;
      background-color: transparent;
      border-bottom: solid black 1px;
      border-radius: 0;
      border: none;
      padding: 8px 0 px 0;
      resize: none;
      
      &:focus {
        outline: none;
      }
    }
```